src/main/java/com/keboola/tableexporter/Application.java

```diff
@@ -45,18 +45,25 @@ private void readConfigFile(String configFile) throws ApplicationException {
             throw new ApplicationException("Configuration file is invalid", ex);
         }
         JSONObject obj = new JSONObject(jsonString);
+        // required params
+        dbUser = obj.getJSONObject("parameters").getJSONObject("db").getString("user");
+        dbPassword = obj.getJSONObject("parameters").getJSONObject("db").getString("#password");
+
+        // database required if not use tnsnames
+        if (tnsnamesPath.equals("")) {
+            dbName = obj.getJSONObject("parameters").getJSONObject("db").getString("database");
+        }

```

Comment:
Dal by som to pod jednu podmienku sem.

https://github.com/keboola/java-oracle-exporter/blob/6dc3bb8bcaf9b066aadcdfa008d8098317963aac/src/main/java/com/keboola/tableexporter/Application.java#L112-L115

Reply:
@webrouse no to jsem chtěl, ale tam zase nemám dostupný config

Reply:
ok, je to cele :spaghetti: 

:slightly_smiling_face: